packages/core/demo/data/CHART_TYPES.ts

```diff
@@ -29,6 +29,11 @@ export default {
 		angular: 'ibm-bullet-chart',
 		vue: 'ccv-bullet-chart',
 	},
+	ChoroplethChart: {
+		vanilla: 'EXPERIMENTAL_ChoroplethChart',
+		angular: 'experimental-ibm-choropleth-chart',
+		vue: 'experimental-ccv-choropleth-chart',
```

Comment:
Can we get someone with experience with vue to verify if this will work. I've tried to run the storybook locally for vue & it states the select `EXPERIMENTAL-ccv-choropleth-chart` doesn't exist.

I'm not too sure how I can set a custom HTML tag for vue 🤔 Might be a storybook issue